src/Components/Layout/Fonts.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { Global } from '@emotion/react'
+
+export const Fonts = (): JSX.Element => (
+  <Global
+    styles={`
+      @font-face {
+        font-family: 'Neue Haas Grotesk Display Pro';
+        src: url('../../../Fonts/NeueHaasDisplay-Roman.eot');
+        src:
+            url('../../../Fonts/NeueHaasDisplay-Roman.eot?#iefix') format('embedded-opentype'),
+            url('../../../Fonts/NeueHaasDisplay-Roman.woff') format('woff'),
+            url('../../../Fonts/NeueHaasDisplay-Roman.ttf') format('truetype');
+        font-weight: normal;
+        font-style: normal;
+      }
+      @font-face {
+        font-family: 'Neue Haas Grotesk Display Pro';
+        src: url('../../../Fonts/NeueHaasDisplay-Bold.eot');
+        src:
+            url('../../../Fonts/NeueHaasDisplay-Bold.eot?#iefix') format('embedded-opentype'),
+            url('../../../Fonts/NeueHaasDisplay-Bold.woff') format('woff'),
+            url('../../../Fonts/NeueHaasDisplay-Bold.ttf') format('truetype');
```

Comment:
Downloaded the font files from [here](https://fontsfree.pro/base-web-fonts/sans-serif-grotesque/940-neue-haas-grotesk.html) - not sure if there's a place people usually get fonts from we can use for other fonts but this seemed to work ok.

Added following [Chakra's second option instructions for adding new fonts](https://chakra-ui.com/guides/using-fonts).